src/gui/src/heatMapRUDY.cpp

```diff
@@ -57,14 +57,17 @@ double RUDYDataSource::getGridXSize() const
     return default_grid_;
   }
 
-  std::vector<int> grid;
-  gCellGrid->getGridX(grid);
+  int max_gridx = 0;
+  for (int i = 0; i < gCellGrid->getNumGridPatternsX(); i++) {
+    int origin, count, step;
+    gCellGrid->getGridPatternX(i, origin, count, step);
+    max_gridx = std::max(max_gridx, step);
```

Comment:
In OR's grt that right most gcell may be wider than the others if the design's width is not a gcell multiple (same with top and height).  I think therefore using the mode would be better than that max.

Reply:
I can fix that.
Should I also fix the code in the GRT heatmap to match as well?